pkg/hookd/database/deployment.go

```diff
@@ -11,6 +11,7 @@ type Deployment struct {
 	Created          time.Time
 	GitHubID         *int
 	GitHubRepository *string
+	GitRefSha        *string
```

Comment:
Not needed unless this value goes into the database, in which case you'll have to add a database migration.